### DISCUSSION TITLE
22621.3527.65.1 Black Screen

### DISCUSSION THREAD

**Discussion by mbsl4y3r** [valinet/ExplorerPatcher#3177] [General]
This release keeps failing and crashing explorer.exe

Tried multiple restarts and multiple installs. If anyone else gets stuck, you need to ctrl+alt+delete and right click on a background process and Open File Location, then either reinstall an older version, for search for explorer.exe in c:\ and keep clicking it until the explorerpatcher.exe error finally pops up and stays disabled and you will be back on w11 normal.

**Comment by Amrsatrio** [comment]
I've pulled the botched prerelease and built a new one that works fine. Sorry for the inconvenience.

**Reply by mbsl4y3r** [reply]
Thank you for all the hard work!
